src/20250814_Multi_GHOInkLaunch/abstraction/constants/GhoCCIPChains.sol

```diff
@@ -0,0 +1,131 @@
+    address ghoAaveCoreSteward;
+    address ghoCCIPSteward;
+    address aclManager;
+    address tokenAdminRegistry;
+    address owner;
+    address ccipRouter;
+  }
+
+  function ARBITRUM() public pure returns (ChainInfo memory) {
+    return
+      ChainInfo({
+        chainSelector: CCIPChainSelectors.ARBITRUM,
+        ghoToken: GhoArbitrum.GHO_TOKEN,
+        ghoCCIPTokenPool: GhoArbitrum.GHO_CCIP_TOKEN_POOL,
+        ghoBucketSteward: GhoArbitrum.GHO_BUCKET_STEWARD,
+        ghoAaveCoreSteward: GhoArbitrum.GHO_AAVE_CORE_STEWARD,
+        ghoCCIPSteward: GhoArbitrum.GHO_CCIP_STEWARD,
+        aclManager: address(AaveV3Arbitrum.ACL_MANAGER),
+        tokenAdminRegistry: CCIPChainTokenAdminRegistries.ARBITRUM,
+        owner: GovernanceV3Arbitrum.EXECUTOR_LVL_1,
+        ccipRouter: CCIPChainRouters.ARBITRUM
+      });
+  }
+
+  function BASE() public pure returns (ChainInfo memory) {
+    return
+      ChainInfo({
+        chainSelector: CCIPChainSelectors.BASE,
+        ghoToken: GhoBase.GHO_TOKEN,
+        ghoCCIPTokenPool: GhoBase.GHO_CCIP_TOKEN_POOL,
+        ghoBucketSteward: GhoBase.GHO_BUCKET_STEWARD,
+        ghoAaveCoreSteward: GhoBase.GHO_AAVE_CORE_STEWARD,
+        ghoCCIPSteward: GhoBase.GHO_CCIP_STEWARD,
+        aclManager: address(AaveV3Base.ACL_MANAGER),
+        tokenAdminRegistry: CCIPChainTokenAdminRegistries.BASE,
+        owner: GovernanceV3Base.EXECUTOR_LVL_1,
+        ccipRouter: CCIPChainRouters.BASE
+      });
+  }
+
+  function ETHEREUM() public pure returns (ChainInfo memory) {
+    return
+      ChainInfo({
+        chainSelector: CCIPChainSelectors.ETHEREUM,
+        ghoToken: GhoEthereum.GHO_TOKEN,
+        ghoCCIPTokenPool: GhoEthereum.GHO_CCIP_TOKEN_POOL,
+        ghoBucketSteward: GhoEthereum.GHO_BUCKET_STEWARD,
+        ghoAaveCoreSteward: GhoEthereum.GHO_AAVE_CORE_STEWARD,
+        ghoCCIPSteward: GhoEthereum.GHO_CCIP_STEWARD,
+        aclManager: address(AaveV3Ethereum.ACL_MANAGER),
+        tokenAdminRegistry: CCIPChainTokenAdminRegistries.ETHEREUM,
+        owner: GovernanceV3Ethereum.EXECUTOR_LVL_1,
+        ccipRouter: CCIPChainRouters.ETHEREUM
+      });
+  }
+
+  function AVALANCHE() public pure returns (ChainInfo memory) {
+    return
+      ChainInfo({
+        chainSelector: CCIPChainSelectors.AVALANCHE,
+        ghoToken: GhoAvalanche.GHO_TOKEN,
+        ghoCCIPTokenPool: GhoAvalanche.GHO_CCIP_TOKEN_POOL,
+        ghoBucketSteward: GhoAvalanche.GHO_BUCKET_STEWARD,
+        ghoAaveCoreSteward: GhoAvalanche.GHO_AAVE_CORE_STEWARD,
+        ghoCCIPSteward: GhoAvalanche.GHO_CCIP_STEWARD,
+        aclManager: address(AaveV3Avalanche.ACL_MANAGER),
+        tokenAdminRegistry: CCIPChainTokenAdminRegistries.AVALANCHE,
+        owner: GovernanceV3Avalanche.EXECUTOR_LVL_1,
+        ccipRouter: CCIPChainRouters.AVALANCHE
+      });
+  }
+
+  function GNOSIS() public pure returns (ChainInfo memory) {
+    return
+      ChainInfo({
+        chainSelector: CCIPChainSelectors.GNOSIS,
```

Comment:
addresses here should be added from address book, for all already in use networks.

Reply:
I agree. Right now this repository only depends on `aave-helpers`, and it's getting the `aave-address-book` dependency from there. I can add `aave-address-book` as an additional direct dependency so we can quickly get its latest version and changes

Reply:
We need this PR to be merged first in order to get the missing Gnosis addresses from the address book
https://github.com/bgd-labs/aave-address-book/pull/1035

Reply:
Now that this was merged, I took addresses from the address book repo. Code change here: https://github.com/bgd-labs/aave-proposals-v3/pull/802/commits/51a56f14c7c6ae0611806730feac2530a3bd785a